spec/services/solr_mapper_spec.rb

```diff
@@ -132,6 +132,19 @@
         expect(solr_mapper.provider_identifier_field).to eq('10.1234/5678')
       end
     end
+
+    context 'with SDR as provider' do
+      let(:metadata) do
+        {
+          provider: 'SDR',
+          identifiers: [{ identifier: '10.1234/5678', identifier_type: 'DRUID' }]
```

Comment:
Can you change this to use a druid instead of a DOI? This is confusing.

Reply:
Yes. Changed it to be more like a druid.